src/bluesearch/entrypoint/database/parse.py

```diff
@@ -14,6 +19,7 @@ def get_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         "parser",
         type=str,
+        choices=("CORD19ArticleParser", "PubmedXMLParser"),
```

Comment:
I agree with you that it's a good idea, the only thing that stopped me before was that this would add a further layer of unnecessary (?) abstraction.
But yes, from the user perspective `cord19-json` is probably much easier to remember than `CORD19ArticleParser`.

Fixed in 82700402c359a1593904556aa286086fb52a2d9d.